openassessment/xblock/data_conversion.py

```diff
@@ -77,6 +77,22 @@ def convert_training_examples_list_to_dict(examples_list):
     ]
 
 
+def list_to_conversational_format(str_list):
+    """
+    String list render method, displaying a list of string values in conversational language.
+    ['a'] => 'a';  ['a', 'b'] => 'a and b';  ['a', 'b', 'c'] => 'a, b, and c'
+
+    Args:
+        str_list (str[]): List of strings to return in comma-joined/conversational form.
+
+    Returns:
+        Combined string.
+    """
+    if len(str_list) < 3:
```

Comment:
I recommend handling the case of str_list is None just to be safe.

Reply:
touche.